README.md

```diff
@@ -11,6 +11,7 @@ This repo contains the content that's used to create the **[Hugging Face course]
 | [Russian](https://huggingface.co/course/ru/chapter1/1) (WIP) | [`chapters/ru`](https://github.com/huggingface/course/tree/main/chapters/ru) | [@pdumin](https://github.com/pdumin) |
 | [Spanish](https://huggingface.co/course/es/chapter1/1) (WIP) | [`chapters/es`](https://github.com/huggingface/course/tree/main/chapters/es) | [@camartinezbu](https://github.com/camartinezbu) |
 | [Thai](https://huggingface.co/course/th/chapter1/1) (WIP) | [`chapters/th`](https://github.com/huggingface/course/tree/main/chapters/th) | [@peeraponw](https://github.com/peeraponw), [@a-krirk](https://github.com/a-krirk), [@jomariya23156](https://github.com/jomariya23156) |
+| [Georgian](https://huggingface.co/course/ka/chapter0/0) (WIP) | [`chapters/ka`](https://github.com/huggingface/course/tree/main/chapters/ka) | [@deepconsc](https://github.com/deepconsc) |
```

Comment:
Currently, the website only renders if `chapter1/1.mdx` is translated. Would you like to add this section to the current PR? If not, I suggest removing this line from the README until that section is translated and then we can add the entry at that point.

Reply:
Hey @lewtun ! 

I'll get at least `chapter1/1.mdx` translated in the following days and will make a new pull request including the changes you requested in the previous comment.

Reply:
Sounds great @deepconsc! Looking forward to seeing this render on the website 🔥